.github/workflows/dependency-review.yml

```diff
@@ -15,6 +15,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: 'Checkout Repository'
-        uses: actions/checkout@v3
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
       - name: 'Dependency Review'
-        uses: actions/dependency-review-action@v1
```

Comment:
wow: I think your changes are correct and I'll approve but... look at where "v1" actually points to: the initial commit event though there are several 1.0.x releases after that! Further proof that actions versioning is really not well thought out.